src/clic-config.adb

```diff
@@ -112,6 +112,23 @@ package body CLIC.Config is
       end if;
    end Get_As_String;
 
+   -------------------
+   -- Get_As_String --
+   -------------------
+
+   function Get_As_String (This    : Instance;
+                           Key     : Config_Key;
+                           Default : String) return String
+   is
+      Val : constant String := Get_As_String (This, Key);
+   begin
+      if Val = "" then
```

Comment:
I think this function should not use the other version of `Get_As_String` but the other way around
```ada
   -------------------
   -- Get_As_String --
   -------------------

   function Get_As_String (This    : Instance;
                           Key     : Config_Key;
                           Default : String) return String
   is
   begin
      if This.Defined (Key) then
         return Image (This.Get (Key).Value);
      else
         return Default;
      end if;
   end Get_As_String;

   -------------------
   -- Get_As_String --
   -------------------

    function Get_As_String (This : Instance;
                            Key : Config_Key)
                            return String
   is (Get_As_String (This, Key, Default => ""));
```